Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1155,7 +1155,8 @@ HolidayNames: !dictionary
     ascensionday: [ hemelvaartsdag ] # 39 Days after Easter
     whitesunday: [ pinksteren ]
     christmas: [ kerstfeest, kerstmis, kerst, xmas ]
-    easterday: [ pasen, paasdag ]
+    easterday: [ pasen, paasdag, paaszondag ]
```

Comment:
`eerste paasdag` should be added, maybe even in favor of paasdag. There is a clear distinction in Dutch between 'eerste' and 'tweede' paasdag. Having paasdag as fallback to easterday is fine by me.

Reply:
added